### PR TITLE
Add unicode functions.

### PR DESCRIPTION
**Pull request by usualwyy**

### CODE REVIEW

File: src/lang/Messages.properties [hunk 1]
```diff
@@ -1094,6 +1094,8 @@ enc2.label.sha1Hash                    = SHA1 Hash
 enc2.label.text                        = Text to be encoded/decoded/hashed
 enc2.label.urlDec                      = URL Decode
 enc2.label.urlEnc                      = URL Encode
+enc2.label.unicode.escapedText         = EscapedText
```
Comment (thc202) [inline src/lang/Messages.properties]: This (and following) should be two words? @kingthorin?

File: src/org/zaproxy/zap/extension/encoder2/EncodeDecodeDialog.java [hunk 2]
```diff
@@ -416,6 +438,77 @@ public String decodeHTMLString(String HTMLText) {
 	public String decodeJavaScriptString(String JavaScriptText) {
 		return StringEscapeUtils.unescapeJavaScript(JavaScriptText);
 	}
+
+	public String encodeUnicodeString(String str) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/encoder2/EncodeDecodeDialog.java]: These (encode/decode method) could be private and static.

File: src/lang/Messages.properties (hunk 1 above)
Comment (kingthorin) [reply src/lang/Messages.properties]: Agreed.

**Comment by usualwyy** [comment]
solved.

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
@usualwyy thanks! How would you like to be credited?
https://github.com/zaproxy/zap-core-help/wiki/HelpCredits

**Pull request merged**